go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -333,10 +333,16 @@ func (oc *Controller) deleteEgressFirewall(egressFirewallObj *egressfirewallapi.
 	deleteDNS := false
 	obj, loaded := oc.egressFirewalls.LoadAndDelete(egressFirewallObj.Namespace)
 	if !loaded {
-		return fmt.Errorf("there is no egressFirewall found in namespace %s", egressFirewallObj.Namespace)
+		return fmt.Errorf("there is no egressFirewall found in namespace %s",
+			egressFirewallObj.Namespace)
 	}
 
-	ef, _ := obj.(egressFirewall)
+	ef, ok := obj.(*egressFirewall)
+	if !ok {
+		return fmt.Errorf("type assertion to *egressFirewall failed for "+
+			" EgressFirewall %s in namespace %s.",
```

Comment:
L343: NIT: you can remove the leading space since you already have trailing space in L342

Reply:
Sure. Done!